src/lib/QueryParams.js

```diff
@@ -1,6 +1,5 @@
-const qParams = {
+/* eslint-disable import/prefer-default-export */
+export const qParams = {
```

Comment:
```suggestion
export default const qParams = {
```

Reply:
Because I include another const `sortParams` I think its needed the way it is unless there is a better way to do it for exporting both const ?

Reply:
Ah, that's because in general it's discouraged to export more than one thing form a file. Hmm, in that case, how about defining `sortParams` at the top of the `FilterBar.jsx` file and exporting it from there? Also I would rename it `sortOptions` since it's not _strictly_ tied to query parameters. The `sortingOptions` dictionary there can be renamed `sortItems` since it's the items needed for the dropdown.

Reply:
Ok done